libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -406,23 +406,21 @@ void Aircraft::set_speedup(float speedup)
 /*
   Create and set in/out socket
 */
-void Aircraft::set_interface_ports(const char* address, const int port_in, const int port_out)
+void Aircraft::set_interface_ports(const char* local_address, const char* gazebo_address, const int port_in, const int port_out)
 {
-    if (!socket_in.bind("127.0.0.1", port_in)) {
-        fprintf(stderr, "SITL: socket in bind failed on sim in : %d  - %s\n", port_in, strerror(errno));
-        fprintf(stderr, "Abording launch...\n");
```

Comment:
I would like to keep fprintf(stderr, "Abording launch...\n"); that way it is sure the launch has quit!